docs/proxy-protocol-sequence-diagram.txt

```diff
@@ -0,0 +1,66 @@
+@startuml
+'Clear Containers Proxy Protocol sequence digram
```

Comment:
- "Clear Container" => "Intel® Clear Containers".
- Typo and missing period: "digram" => "diagram."